### PR TITLE
Improving .gitignore

### PR DESCRIPTION
**Pull request by RyDroid**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,35 @@
+# Copying and distribution of this file, with or without modification,
```
Comment (brarcher) [inline .gitignore]: If you would like to put a license, I would prefer the same license as is in the other files of the project, namely the LGPLv2

**Comment by RyDroid** [comment]
I removed the license notice.

**Comment by brarcher** [comment]
Could you now update this branch, so it can up merged in?

Jenkins: ok to test

**Comment by RyDroid** [comment]
Ok, I did it.

**Pull request merged**